util/network/netproviders/network.go

```diff
@@ -35,16 +38,21 @@ func Providers(opt Opt) (map[pb.NetMode]network.Provider, error) {
 			}
 			defaultProvider = cniProvider
 		} else {
-			logrus.Warnf("using host network as the default")
-			defaultProvider = network.NewHostProvider()
+			defaultProvider = getFallback()
 		}
 	default:
 		return nil, errors.Errorf("invalid network mode: %q", opt.Mode)
 	}
 
-	return map[pb.NetMode]network.Provider{
+	providers := map[pb.NetMode]network.Provider{
 		pb.NetMode_UNSET: defaultProvider,
-		pb.NetMode_HOST:  network.NewHostProvider(),
 		pb.NetMode_NONE:  network.NewNoneProvider(),
-	}, nil
+	}
+
+	hostProvider, err := getHostProvider()
```

Comment:
I'd rather return a `bool` here if we are going to ignore the error. So that if there is a need for actual errors in the future it needs to be added and will not be ignored then.

Reply:
Yeah, makes sense, and easy to fix in both places.